packages/core/src/model/chain/redditArbitrum.ts

```diff
@@ -0,0 +1,21 @@
+import { Chain } from '../../constants'
+import { getAddressLink, getTransactionLink } from '../../helpers/chainExplorerLink'
+
+const testnetRedditscanUrl = 'https://testnet.redditspace.com'
+
+export const RedditTestnet: Chain = {
+  chainId: 5391184,
+  chainName: 'RedditTestnet',
+  isTestChain: true,
+  isLocalChain: false,
+  multicallAddress: '0xFf79F208541DdF9dA6632dD851a269BbfA97d9aD',
+  multicall2Address: '',
```

Comment:
Can you deploy multicall2?

Reply:
Yup the multicall2 has been deployed at: `0xd4d664d419a6a845c98cc366ae1c4b24592bd5ce`